x-pack/test/alerting_api_integration/spaces_only/tests/alerting/builtin_alert_types/index_threshold/create_test_data.ts

```diff
@@ -53,6 +53,7 @@ async function createEsDocument(es: any, epochMillis: number, testedValue: numbe
     source: DOCUMENT_SOURCE,
     reference: DOCUMENT_REFERENCE,
     date: new Date(epochMillis).toISOString(),
+    date_epoch_millis: epochMillis,
```

Comment:
added a new date field, but in epoch millis format, to test using it as a time field